GitCommands/Git/GitModule.cs

```diff
@@ -1485,7 +1486,16 @@ public bool ResetChanges(ObjectId? resetId, IReadOnlyList<GitItemStatus> selecte
             }
 
             output.Append(CheckoutIndexFiles(filesToReset));
-            CheckoutFiles(filesToCheckout, resetId, force: false);
+            output.Append(CheckoutFiles(filesToCheckout, resetId, force: false));
+            if (filesInUse.Count > 0)
```

Comment:
1. DRY
2. The MessageBox breaks lines at the space after the bullet. This has confused me.
```suggestion
            const char bullet = `\u2022';
            const char noBreakSpace = '\u00a0';
            const string separator = $"{Environment.NewLine}{bullet}{noBreakSpace}";
            if (filesInUse.Count > 0)
```